kernel/kernel-api/src/main/java/io/delta/kernel/client/FileHandler.java

```diff
@@ -17,7 +17,7 @@
 package io.delta.kernel.client;
 
 import io.delta.kernel.data.Row;
-import io.delta.kernel.expressions.Expression;
+import io.delta.kernel.expressions.predicates.Predicate;
```

Comment:
do we really need the `predicates` package? The more nested packages we have .. more importing for all the users.

Reply:
Otherwise `import io.delta.kernel.expressions.*` would be good every where.

Reply:
awesome, removing the package.